homeassistant/components/homewizard/number.py

```diff
@@ -0,0 +1,69 @@
+"""Creates HomeWizard Number entities."""
+from __future__ import annotations
+
+from homeassistant.components.number import NumberEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import PERCENTAGE
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import EntityCategory
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import HWEnergyDeviceUpdateCoordinator
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up numbers for device."""
+    coordinator: HWEnergyDeviceUpdateCoordinator = hass.data[DOMAIN][entry.entry_id]
+
+    if coordinator.data["state"]:
+        async_add_entities(
+            [
+                HWEnergyNumberEntity(coordinator, entry),
+            ]
+        )
+
+
+class HWEnergyNumberEntity(
+    CoordinatorEntity[HWEnergyDeviceUpdateCoordinator], NumberEntity
+):
+    """Representation of status light number."""
+
+    _attr_entity_category = EntityCategory.CONFIG
+    _attr_has_entity_name = True
+
+    def __init__(
+        self,
+        coordinator: HWEnergyDeviceUpdateCoordinator,
+        entry: ConfigEntry,
+    ) -> None:
+        """Initialize the control number."""
+        super().__init__(coordinator)
+        self._attr_unique_id = f"{entry.unique_id}_status_light_level"
+        self._attr_name = "Light level"
```

Comment:
I think that calling it Light level seems confusing. Shouldn’t it be something more like Status light brightness ?

Reply:
Agreed, feels better. Fixed in https://github.com/home-assistant/core/pull/82378/commits/7dfa55bc3ceedbbfe195115e87e89cca830f6e0b